dict/indonesian.yml

```diff
@@ -0,0 +1,292 @@
+# Created by Kohei Watanabe for the newsmap package: https://github.com/koheiw/newsmap
+# Indonesian version was created by Hiroko Kinoshita 
+
+AFRICA:
+  EAST:
+    'BI': [Burundi, Burundi*, Orang Brundi, Bujumbura]
+    'DJ': [Djibouti, Djibouti*, Orang Djibouti]
+    'ER': [Eritrea, Eritrea*, Orang Eritrea, Asmara]
+    'ET': [Etiopia, Etiopia*, Orang Etiopia, Addis Ababa]
+    'KE': [Kenya, Kenya*, Orang Kenya, Nairobi]
+    'KM': [Komoro, Komoro*, Orang Komoro, Moroni]
+    'MG': [Madagaskar, Madagaskar*, Orang Madagaskar, Antananarivo]
+    'MU': [Mauritius, Mauritius*, Orang Mauritius, Port Louis]
+    'MW': [Malawi, Malawi*, Orang Malawi, Lilongwe]
+    'MZ': [Mozambik, Mozambik*, Orang Mozambik, Maputo]
+    'RE': [Reunion, Réunion, Reunion*, Réunion*, Orang Réunionese, Orang Reunionese, Reunionnais]
+    'RW': [Rwanda, Rwanda*, Orang Rwanda, Kigali]
+    'SC': [Seychelles, Seychelles*, Orang Seychelloes, Seychellois]
+    'SO': [Somalia, Somalia*, Orang Somalia,  Mogadishu]
+    'TZ': [Tanzania, Tanzania*, Orang Tanzania, Dodoma, Dar es Salaam]
+    'UG': [Uganda, Uganda*, Orang Uganda, Kampala]
+    'YT': [Mayotte, Mayotte*, Orang Mayotte, Mamoudzou]
+    'ZM': [Zambia, Zambia*, Orang Zambia, Lusaka]
+    'ZW': [Zimbabwe, Zimbabwe*, Orang Zimbabwe, Harare]
+
+  MIDDLE:
+    'AO': [Angola, Angola*, Orang Angola, Luanda]
+    'CD': [Republik Demokratik Kongo, DR Kongo, DRK, Demockratik Republik Kongo, DR Kongo*, Orang Republik Demokratik Kongo, Kinshasa]
+    'CF': [Republik Afrika Tengah, Afrika Tengah, Republik Afrika Tengah*, Afrika Tengah*, Orang Republik Afrika Tengah, Orang Afrika Tengah, Bangui]
+    'CG': [Kongo, Republik Kongo, Kongo*, Republik Kongo*, Orang Kongo, Orang Republik Kongo, Brazzaville]
+    'CM': [Kamerun, Kamerun*, Orang Kamerun, Orang Yaounde, Yaoundé]
+    'GA': [Gabon, Gabon*, Orang Gabon, Libreville]
+    'GQ': [Guinea Khatulistiwa, Guinea Ekuatorial, Guinea Khatulistiwa*, Guinea Ekuatorial*, Orang Guinea Khatulistiwa, Orang Ekuatoguinea, Malabo]
+    'ST': [Sao Tome dan Principe, São Tomé und Príncipe, Sao Tome dan Principe*, São Tomé und Príncipe*, Orang Sao Tome dan Principe]
+    'TD': [Chad, Chad*, Orang Chad, N'Djamena]
+
+  NORTH:
+    'DZ': [Aljazair, Aljazair*, Orang Aljazair, Aljir]
+    'EG': [Mesir, Maeir*, Orang Mesir, Kairo]
+    'EH': [Sahara Barat, Sahara Barat*, Orang Sahara Barat, El Aaiun]
+    'LY': [Libya, Libia, Libya*, Libia*, Orang Libya, Orang Libia, Tripoli]
+    'MA': [Maroko, Maroko*, Orang Maroko, Rabat]
+    'SD': [Sudan, Sudan*, Orang Sudan, Khartoum]
+    'SS': [Sudan Selatan, S Sudan, Sudan Seletan*, Orang Sudan Selatan, Juba]
```

Comment:
@hirokokinoshita should we have "Sudan S" instead of "S Sudan"?

Reply:
I would like to keep it because the word order is reversed in Indonesian language and also English is used frequently.